org.opent2t.sample.thermostat.superpopular/com.smartthings.thermostat/js/thingTranslator.js

```diff
@@ -410,6 +595,52 @@ class Translator {
         subscriptionInfo.endpointUri = this.endpointUri;
         return this.smartThingsHub._unsubscribe(subscriptionInfo);
     }
+
+    _resourceSchemaToProviderSchemaAsync(resourceId, resourceSchema) {
+
+        // build the object with desired state
+        var result = {};
```

Comment:
by doing Object.create(null) you won't get any of the basic functions attached to the object whereas with {} you do. depends on your use-case

Reply:
Keeping consistent with other translators, will add to code clean up task.